source/community.applications/usr/local/emhttp/plugins/community.applications/skins/Narrow/skin.html

```
@@ -1,113 +1,528 @@
-  <div class='alternateViewContent'></div>
+<div id="caViewportBlocker" class="ca_viewportBlocker ca_hide" aria-hidden="true"></div>
+<div class='mobileMenu' id='mobileMenu'>
+		<div class='menuItems menuAdjust'>
+			<ul class='caMenu '>
+				<li class='startupButton caMenuItem onlyShowWithFeed' data-category='new'>
+					<?tr("Home");?>
+					<div id='ca_homeSectionSubtitle' class='ca_homeSectionSubtitle ca_hide'></div>
+					<div id='ca_homeSearchSubtitle' class='ca_homeSearchSubtitle ca_hide'></div>
+				</li>
+				<li><hr class='onlyShowWithFeed'/></li>
+				<li class='caMenuItem sectionMenu onlyShowWithFeed' data-category='installed_apps'><?tr("Installed Apps");?>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)